packages/react/src/utils/requestLock.ts

```diff
@@ -0,0 +1,28 @@
+export function requestLock(id: string, cb: (id: string) => void) {
```

Comment:
❓ Did we add locks support now or did we have it already in @novu/js? Actually I am a bit surprised this is not in @novu/js

Reply:
@SokratisVidros We have it for the JS package. However, we needed to copy this code for the `useWebSocketEvent` hook implementation. We can actually expose it from the JS package as part of the `@novu/js/internal` export or smth similar.